akka-cluster/src/test/scala/akka/cluster/StartupWithOneThreadSpec.scala

```diff
@@ -60,7 +60,6 @@ class StartupWithOneThreadSpec(startTime: Long) extends AkkaSpec(StartupWithOneT
       system.actorOf(testProps) ! "hello"
       system.actorOf(testProps) ! "hello"
 
-      val cluster = Cluster(system)
```

Comment:
this was very much part of the test, change to 
```
Cluster(system)
```

Reply:
Doh